frontend/src/features/env/queries.ts

```diff
@@ -43,3 +46,15 @@ export const usePublicFeedbackFormView = (
     { enabled },
   )
 }
+
+type UseGlobalBetaReturn = {
+  flagEnabled?: boolean
```

Comment:
I think we should drop the prefix `flag`, it's clear from the function name and context that we're getting a flag by name, so just `enabled` should do.

Reply:
Done in [d8c8e1b](https://github.com/opengovsg/FormSG/pull/6286/commits/d8c8e1b640c2e0be856bd833dfc48d9a43ad09f8)